option/option.go

```diff
@@ -0,0 +1,61 @@
+package option
+
+// Config holds configuration options for YAML processing
+type Config struct {
```

Comment:
```suggestion
type config struct {
```

This type probably does not need to be public.